src/modules/size-controls/size-controls.css

```diff
@@ -0,0 +1,37 @@
+@import '../core/core.css';
+
+size-controls .button {
+    display: none;
+    margin-right: var(--default-margin);
+}
+
+.size-controls-fieldset {
+    display: none;
+}
+
+.size-controls-input {
+    height: 42px; /* line-height has no effect on replaced elements */
+    width: 6em;
+    box-sizing: border-box;
+    padding: 0 1em;
+    background: var(--color-grey-light);
+    border: 1px solid transparent;
+}
+
+.size-controls-input:hover {
+    transition: all 0.2s ease-in;
+    border: 1px solid var(--color-blue-dark);
+    background-color: var(--color-grey);
+}
+
+@media screen and (min-width: 525px) {
```

Comment:
I think showing or hiding the "size-controls" makes more sense to define on the level of the app. The app knows when they will fit or not in relation to other content of the top toolbar.